README.md

```diff
@@ -32,6 +32,9 @@ Requirements
 ============
 - chef >= 10.14
```

Comment:
I think it makes sense to update this to `12.7` as well (so that `master` reflects the support of the latest release)

Reply:
Absolutely (this has already been covered in #450 which is the one actually dropping the support for < 12.7)

Reply:
oh well, thanks, disregard then :)